config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
   template:
     spec:
       containers:
-        - image: docker.io/linode/cluster-api-provider-linode:main
+        - image: docker.io/linode/cluster-api-provider-linode:v0.1.0
```

Comment:
I think we want to keep this as main, our release workflow overwrites this to the exact tag of the release but this default tag is pushed on every merge to main